pvlib/irradiance.py

```diff
@@ -515,7 +550,7 @@ def get_ground_diffuse(surface_tilt, ghi, albedo=.25, surface_type=None):
         (e.g. surface facing up = 0, surface facing horizon = 90).
 
     ghi : numeric
-        Global horizontal irradiance in W/m^2.
+        Global horizontal irradiance. [W/m^2]
 
     albedo : numeric, default 0.25
         Ground reflectance, typically 0.1-0.4 for surfaces on Earth
```

Comment:
We can do better than this, pvsyst gives a rough table here: https://www.pvsyst.com/help/albedo.htm then copied verbatim into [SolarFarmer](https://dnvgldocs.azureedge.net/SolarFarmer_Latest/CalcRef/Theory/Transposition.html#ground-albedo-irradiance) and [pvpmc](https://pvpmc.sandia.gov/modeling-steps/1-weather-design-inputs/plane-of-array-poa-irradiance/calculating-poa-irradiance/poa-ground-reflected/albedo/). At the very least we can give snow (0.6-0.8), dirt and grass (0.2-0.25) and water (0.1). I think we should raise a new issue for this though, I wouldn't try to change it here in this PR, it's unrelated. 😉

Reply:
Agree. Edits are creeping outwards e.g. the unit notation in `get_ground_diffuse`. I'll resist the urge.